[2022-09-05 03:07:39,004] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task 2022-08-12T00:00:00+00:00 [queued]>
[2022-09-05 03:07:39,189] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task 2022-08-12T00:00:00+00:00 [queued]>
[2022-09-05 03:07:39,194] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-09-05 03:07:39,194] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2022-09-05 03:07:39,195] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-09-05 03:07:39,280] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2022-08-12T00:00:00+00:00
[2022-09-05 03:07:39,286] {standard_task_runner.py:52} INFO - Started process 311 to run task
[2022-09-05 03:07:39,341] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v2', 'second_task', '2022-08-12T00:00:00+00:00', '--job-id', '45', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpbxffj7v1', '--error-file', '/tmp/tmpz3jxfnjz']
[2022-09-05 03:07:39,372] {standard_task_runner.py:77} INFO - Job 45: Subtask second_task
[2022-09-05 03:07:39,604] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v2.second_task 2022-08-12T00:00:00+00:00 [running]> on host 2b8e628fc16c
[2022-09-05 03:07:39,813] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v2
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-08-12T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-12T00:00:00+00:00
[2022-09-05 03:07:39,819] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2022-09-05 03:07:39,822] {bash.py:158} INFO - Running command: echo hey, I am task2 and will be running after task1!
[2022-09-05 03:07:39,872] {bash.py:169} INFO - Output:
[2022-09-05 03:07:39,875] {bash.py:173} INFO - hey, I am task2 and will be running after task1!
[2022-09-05 03:07:39,876] {bash.py:177} INFO - Command exited with return code 0
[2022-09-05 03:07:40,161] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_task, execution_date=20220812T000000, start_date=20220905T030739, end_date=20220905T030740
[2022-09-05 03:07:40,586] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-09-05 03:07:40,628] {local_task_job.py:146} INFO - Task exited with return code 0
